views/vulns.erb

```diff
@@ -0,0 +1,156 @@
+
+        <div class="control">
+          <select class="select" name="lfi[filter_bypass]">
+            <option value="" />
+            <option value="nullbyte">nullbyte</option>
+            <option value="double_escape">double_escape</option>
+            <option value="base64">base64</option>
+            <option value="rot13">rot13</option>
+            <option value="zlib">zlib</option>
+          </select>
+        </div>
+      </div>
+
+      <h3>RFI</h3>
+
+      <div class="field">
+        <label class="label">Filter Bypass</label>
+
+        <div class="control">
+          <select class="select" name="rfi[filter_bypass]">
+            <option value="" />
+            <option value="double_encode">double_encode</option>
+            <option value="suffix_escape">suffix_escape</option>
+            <option value="null_byte">null_byte</option>
+          </select>
+        </div>
+      </div>
+
+      <div class="field">
+        <label class="label">Test Script URL</label>
+
+        <div class="control">
+          <% if @errors && @errors[:rfi]&.dig(:test_script_url) %>
+            <input class="input is-danger" type="text" name="rfi[test_script_url]" value="<%=hattr params.dig(:rfi, :test_script_url)%>">
+
+            <% @errors[:rfi][:test_script_url].each do |error| %>
+              <p class="help is-danger"><%=h error %></p>
+            <% end %>
+          <% else %>
+            <input class="input" type="text" name="rfi[test_script_url]" value="<%=hattr params.dig(:rfi, :test_script_url) %>">
+          <% end %>
+        </div>
+      </div>
+
+      <h3>SQLI</h3>
+
+      <div class="field">
+        <label class="has-text-weight-semibold">Escape Quote: </label>
+        <input class="checkbox" type="checkbox" name="sqli[escape_quote]"<%= " checked" if params.dig(:sqli, :terminate) %>
+      </div>
+
+      <div class="field">
+        <label class="has-text-weight-semibold">Escape Parens: </label>
+        <input class="checkbox" type="checkbox" name="sqli[escape_parens]"<%= " checked" if params.dig(:sqli, :terminate) %>
+      </div>
+
+      <div class="field">
+        <label class="has-text-weight-semibold">Terminate: </label>
+        <input class="checkbox" type="checkbox" name="sqli[terminate]"<%= " checked" if params.dig(:sqli, :terminate) %>
+      </div>
+
+      <h3>SSTI</h3>
+
+      <div class="field">
+        <label class="label">Escape</label>
+
+        <div class="control">
+          <select class="select" name="ssti[escape]">
+            <option value="" />
+            <option value="double_curly_braces">double_curly_braces</option>
+            <option value="dollar_curly_braces">dollar_curly_braces</option>
+            <option value="dollar_double_curly_braces">dollar_double_curly_braces</option>
+            <option value="pound_curly_braces">pound_curly_braces</option>
+            <option value="angle_brackets_percent">angle_brackets_percent</option>
```

Comment:
I think we should add the above code to add an optional `selected` attribute if the param is set.

```erb
<%= " checked" if params.dig(:ssti, :escape)  == 'angle_brackets_percent' %>
```